src/Arcus.Template.WebApi/Arcus.Template.WebApi.csproj

```diff
@@ -2,6 +2,9 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.2</TargetFramework>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <DocumentationFile>Arcus.Template.WebApi.Open-Api.xml</DocumentationFile>
+    <!--#if (AuthoringMode)-->
```

Comment:
Is this intentional?

Reply:
Yep, all the tags inside the `AuthoringMode` will be removed when the project is created.

We don't all those tags with Arcus info for example.

Reply:
Perfect, just wanted to make sure :D